src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
please fix your calendar according to requirements:
![image](https://github.com/mate-academy/layout_calendar/assets/102289654/77f611a3-f333-4dca-823a-9788bb0ef4e2)